modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1790,11 +1790,13 @@ return view.extend({
 		s.handleJoinConfirm = function(radioDev, bss, form, ev) {
 			var nameopt = L.toArray(form.lookupOption('name', '_new_'))[0],
 			    passopt = L.toArray(form.lookupOption('password', '_new_'))[0],
+			    ssidopt = L.toArray(form.lookupOption('ssid', '_new_'))[0],
```

Comment:
ssid or essid ?

Reply:
However you named the option - `'ssid'` in your case.